src/Model/InteractiveCampaign.php

```diff
@@ -138,23 +180,22 @@ public function relevantInteractives($url = null, $page = null)
     }
 
     /**
-     * Is this campaign viewable? Checks start / expires dates
-     *
-     * @param type $url
-     * @param type $pageType
+     * Is this campaign active now? Checks start / expires dates
      */
-    public function viewableOn($url, $pageType = null)
```

Comment:
This naming was to be consistent between Campaign and Interactive logic - can you confirm that's unaffected?

Reply:
Just had a quick look - InteractiveControllerExtension calls `$campaign->invokeWithExtensions('viewableOn', $url, $class);` so this may be best leaving the method name as it was

Reply:
Yea, 'now' is more accurate and serves I gathered the 'on' methods should be left as-is.